content/en/docs/contribute/advanced.md

```diff
@@ -86,6 +86,7 @@ Responsibilities for New Contributor Ambassadors include:
 - Mentoring new contributors through their first few PRs to the docs repo.
 - Helping new contributors create the more complex PRs they need to become Kubernetes members.
 - [Sponsoring contributors](/docs/contribute/advanced/#sponsor-a-new-contributor) on their path to becoming Kubernetes members.
+- Hosting a monthly meeting with an existing SIG Docs contributor for new contributors.
```

Comment:
It's a little unclear to me - does the NCA organize a meeting that an existing contributor attends, so that new contributors can ask questions?

Reply:
```suggestion
- Hosting a monthly meeting to help and mentor new contributors.
```
How's this change? The bullets above state the NCA mentors and helps new contributor and the monthly meeting can be a virtual venue for this.

Reply:
+1 for Rey's suggestion :)

Reply:
Ah yes thank you that's much clearer! 😄